src/Symfony/Component/Security/Core/Authorization/ExpressionLanguage.php

```diff
@@ -13,23 +13,27 @@
 
 use Symfony\Component\ExpressionLanguage\ExpressionLanguage as BaseExpressionLanguage;
 
-/**
- * Adds some function to the default ExpressionLanguage.
- *
- * @author Fabien Potencier <fabien@symfony.com>
- *
- * @see ExpressionLanguageProvider
- */
-class ExpressionLanguage extends BaseExpressionLanguage
-{
+if (!class_exists(BaseExpressionLanguage::class)) {
+    throw new \RuntimeException(sprintf('The "%s" class requires the "ExpressionLanguage" component. Install "symfony/expression-language" to use it.', ExpressionLanguage::class));
```

Comment:
👍 for this. To make the language consistent with other spots:

> The "%s" class requires the "ExpressionLanguage" component. Try running "composer require symfony/expression-language".

Is there a specific time when this class is used? I mean, obviously, if you reference the class directly, it's used :). But is it if you try to use expressions inside, for example, `access_control`? I'm asking only because if there is a specific use-case that activates this class 99% of the time, it might be better to mention that use case (e.g. "To use expressions in access_control, you must install...") versus saying that that `ExpressionLanguage` class requires the component (when the user is not really interacting with this class directly, so might not be sure where this is coming from)